Demo File

```diff
@@ -0,0 +1 @@
+Demo File
```

Comment:
add a line comment to the Pull Request mentioning that you should add your favorite color